.travis.yml

```diff
@@ -1,13 +1,27 @@
 language: java
 sudo: false
-install: true
 jdk: oraclejdk8
-script: ./travis.sh
+install: ./gradlew build
+script:
+  - ./gradlew javadoc
+  - ./sonar.sh
+after_success: ./dropbox.sh
 before_cache:
-- rm -rf $HOME/.gradle/caches/modules-2/modules-2.lock
```

Comment:
No, we want the folder cached and only delete the stuff that changes every build